ibis/expr/api.py

```diff
@@ -192,6 +193,7 @@
     'param',
     'pi',
     'prevent_rewrite',
+    'psql',
```

Comment:
How about calling is `pretty_sql`? 

`psql` is a bit cryptic for me, first I associated to postgres.

Reply:
I want it to be short to type :) Ideally <= 5 characters and no underscores.

Reply:
I'd settle for `pretty` I guess.

Reply:
Not everything in ibis is sql though, I think sql has to be in the name. How about `tosql`? Or `viewsql`? Idk, don't want to bikeshed here.

Reply:
How about `prsql`?

Reply:
Or just `sql()`? 

`from ibis.common.pretty import sql`

But we can always alias an import.

Reply:
I'm going with `showsql`. `sql` is too generic.